appinventor/appengine/src/com/google/appinventor/client/style/neo/NewYoungAndroidProjectWizardNeo.ui.xml

```diff
@@ -0,0 +1,41 @@
+<!-- Copyright 2009-2011 Google, All Rights reserved -->
+<!-- Copyright 2011-2023 MIT, All rights reserved -->
+<!-- Released under the Apache License, Version 2.0 -->
+<!-- http://www.apache.org/licenses/LICENSE-2.0 -->
+
+<!-- As of Apr 2024, this template is not being used. CSS changes are handling all the changes
+to dialog we need currently. It is left here as an example of a dialog template in case we want
+to do that in the future. -->
```

Comment:
Maybe put this into documentation instead rather than leaving vestigial code that was never used in the source tree.

Reply:
> Maybe put this into documentation instead rather than leaving vestigial code that was never used in the source tree.

Yeah, this probably isn't the best way to handle it. This is just such a good example to use for future work when we actually do want to do something with a dialog that requires a template. We should discuss where it should go in documentation so that it's easily discoverable.